Sources/SpeziLLMLocal/LLMLocalPlatform.swift

```diff
@@ -59,34 +57,22 @@ public actor LLMLocalPlatform: LLMPlatform, DefaultInitializable {
         self.init(configuration: .init())
     }
     
-    
     public nonisolated func configure() {
-        // Initialize the llama.cpp backend
-        llama_backend_init()
-        llama_numa_init(configuration.nonUniformMemoryAccess.wrappedValue)
+#if targetEnvironment(simulator)
+        assertionFailure("SpeziLLMLocal: Code cannot be run on simulator.")
+#endif
```

Comment:
Just to clarify; this will never be able to work on the simulator due to an MLX limitation?

If that's the case, we should add a good mock-infrastructure for an LLM within SpeziLLM and might want to use that in SpeziLLMLocal in this case to enable testing and prototyping on a simulator. Might be worth a well-documented issue.

Reply:
Yes, I just confirmed it once again with a fresh project and just MLX imported, the app crashes (Version `mlx-swift:0.18.0`).
I will think about a good testing strategy and include it in the README in a future commit.

<img width="600" alt="image" src="https://github.com/user-attachments/assets/5b5dcd4f-8e52-4759-b61a-a964191389d9">

Reply:
A comment in the README would be great 🚀

As noted, we should ensure that we have a proper mock LLM infrastructure in there.

Reply:
@LeonNissen Would be great if we can create a small issue in this repo to aim to improve the mocking infrastructure in SpeziLLM local in case people want to develop on the simulator; would be great to have a nice mechanism to just use elements from our other mocking elements in SpeziLLM.